prov/efa/test/efa_unit_test_ep.c

```diff
@@ -127,3 +127,40 @@ void test_rxr_ep_dc_atomic_error_handling(struct efa_resource **state)
 	/* make sure there is no leaking of tx_entry */
 	assert_true(dlist_empty(&rxr_ep->tx_entry_list));
 }
+
+/**
+ * @brief verify that when shm was used to send small message (<4k), no copy should be perform.
```

Comment:
typo `Verify that when shm is used to send small messages (<4k), no copy should be performed.`